WaterLog_Backend/Procedures.cs

```diff
@@ -1006,6 +1006,13 @@ public enum Period
             arrayOfSeasonsCost.AddPoint("Autum", cost_season[3]);
             return arrayOfSeasonsCost;
         }
+
+        public async Task<double> getTankLevel(int Id)
+        {
+
```

Comment:
Is this function supposed to be like this? 
also remove empty lines in the function please

Reply:
Tank level function no longer exists.